server/src/utils/adjustFollowingDeliveryPhases.js

```diff
@@ -0,0 +1,44 @@
+const { db } = require('../db/models');
+
+module.exports = (model) => {
+  const newModel = model;
+  const nextUnfrozenPhase = async () => {
+    if (newModel.itemInFrontOf) {
+      const nextPhase = await db.deliveryPhase.findOne({
+        where: { id: newModel.itemInFrontOf },
+      });
+      if (nextPhase.freezeDuration) {
+        if (nextPhase.itemInFrontOf) {
+          newModel.itemInFrontOf = nextPhase.itemInFrontOf;
+          nextUnfrozenPhase(newModel);
+        } else {
+          nextPhase.itemInFrontOf = newModel.id;
+          await db.deliveryPhase.upsert(nextPhase.dataValues);
+          newModel.itemInFrontOf = null;
+          await db.deliveryPhase.upsert(newModel);
+        }
+      }
+      if (!nextPhase.freezeDuration) {
+        if (newModel.durationInHours >= nextPhase.durationInHours) {
+          await db.deliveryPhase.destroy({
+            where: nextPhase.dataValues,
+          });
+        } else {
+          const newNextPhaseDuration =
+            nextPhase.durationInHours - newModel.durationInHours;
```

Comment:
For stuff like this I think it's actually more readable to do it in one line and get rid of the const

Reply:
removed const